specification/paths/RegisterMultiColliShipment.json

```
@@ -142,7 +142,31 @@
                   "service_option_ids": {
                     "type": "array",
                     "items": {
-                      "$ref": "#/components/schemas/Uuid"
+                      "type": "object",
+                      "required": [
+                        "id",
+                        "values"
```

This is actually not required! Most service options do not have values_format!
```suggestion
                        "id"
```